### DISCUSSION TITLE
Man kan ikke oprette lokation

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#31] [Bugs] [answered]
Man kan ikke oprette en ny lokation og lokationen gemmes ikke. 

Jeg har netop prøvet fra mac og PC at oprette en ny lokation for arrangøren Grønttorvet.

**Comment by mikkelsl** [comment, marked as answer]
Har lavet en opgave i Development: https://github.com/xooom-community/product/issues/204

**Comment by henrikbak** [comment]
@MartinQwist Kan du give lidt mere information om denne? Jeg kan ikke umiddelbart fremprovokere fejlen.

- I hvilen forbindelse opretter du en ny lokation?
- Hvad indtaster du i formularen?
- Får du en fejlmeddelese?

Du kan evt sende et screenshot.
